NewSpring/ApollosProject
# [DEV-3838] Fixes Android crash when opening LocationFinder

## DESCRIPTION

### What does this PR do, or why is it needed?

Currently, when in the onboarding LocationFinder or in the setting Locations view, it crashes on Android. Google Maps Platform recently required projects using the specifications that we use to include a reference to a legacy library, documentation [here](https://developers.google.com/maps/documentation/android-sdk/config#specify_requirement_for_apache_http_legacy_library).

Also, there is a duplicate reference to the `react-native-maps` library in the settings.gradle, so I removed it.

### How do I test this PR?

On Android, navigate to a mapview and ensure that the app doesn't crash.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
